.github/workflows/nightly.yaml

```diff
@@ -0,0 +1,356 @@
+name: nightly
```

Comment:
is it my impression or we do the same things with the `ci.yaml` right? Why we are not adding the schedule on `ci.yaml`  and it will run when is needed? We did a similar thing to plugins

Reply:
Yep . Plus we are running e2e tests and we do not run coverage / dist . I implemented this in the first place but I did not like the conditions . I will merge them though you are right

Reply:
Actually another reason I 've separated those is to easily distinguish them from the Github Web UI . If we merge them we will have to distinguish them with filters